javascript/index.js

```diff
@@ -32,14 +52,26 @@ function printMilliseconds() {
 
 function printSplit() {
   // ... your code goes here
+  var childElement = document.createElement("LI");
+  childElement.innerHTML = chronometer.splitClick()
+  splits.appendChild(childElement);
+
 }
 
 function clearSplits() {
   // ... your code goes here
+ 
+  while (splits.hasChildNodes()) {  
+    splits.removeChild(splits.firstChild);
+  }
+  
+  
 }
 
 function setStopBtn() {
   // ... your code goes here
+  chronometer.stopClick;
```

Comment:
On line 104, you're already firing `chronometer.stopClick()` so you don't need this line here.

But you could have used function setStopBtn for DOM manipulation
```
function setStopBtn() {
  btnLeft.className = 'btn stop';
  btnLeft.textContent = 'STOP';
}